### PR TITLE
chore: feature flag removal of accept-invite

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/entryPoints/AcceptInvite.res [hunk 1]
```diff
@@ -75,11 +74,7 @@ let make = () => {
         ]->getJsonFromArrayOfJson
       let res = await updateDetails(url, body, Post, ())
       let email = HSLocalStorage.getFromMerchantDetails("email")
-      let token = HyperSwitchAuthUtils.parseResponseJson(
-        ~json=res,
-        ~email,
-        ~isAcceptInvite=acceptInvite,
-      )
+      let token = HyperSwitchAuthUtils.parseResponseJson(~json=res, ~email, ~isAcceptInvite=true)
```
Comment (Riddhiagrawal001) [inline src/entryPoints/AcceptInvite.res:77]: `~isAcceptInvite=true` This is true in all the places so do we need this param ?

File: src/screens/SwitchMerchant/SwitchMerchant.res [hunk 2]
```diff
@@ -285,7 +284,7 @@ let make = (~userRole, ~isAddMerchantEnabled=false) => {
       let token = HyperSwitchAuthUtils.parseResponseJson(
         ~json=res,
         ~email=responseDict->LogicUtils.getString("email", ""),
-        ~isAcceptInvite=acceptInvite,
+        ~isAcceptInvite=true,
```
Comment (Riddhiagrawal001) [inline src/screens/SwitchMerchant/SwitchMerchant.res:287]: same here

File: src/screens/APIUtils/APIUtils.res [hunk 3]
```diff
@@ -176,9 +176,7 @@ let getURL = (
     | #ACCEPT_INVITE => `${userUrl}/user/invite/accept`
     | #USER_DELETE => `${userUrl}/user/delete`
     | #UPDATE_ROLE => `${userUrl}/user/${(userType :> string)->String.toLowerCase}`
-    | #SIGNIN
```
Comment (Riddhiagrawal001) [inline src/screens/APIUtils/APIUtils.res:179]: Can we verify that we dont need these anymore in any case

Comment (PritishBudhiraja) [reply src/screens/APIUtils/APIUtils.res:183]: Yes. We don't need

File: src/screens/SwitchMerchant/SwitchMerchant.res (hunk 2 above)
Comment (PritishBudhiraja) [reply src/screens/SwitchMerchant/SwitchMerchant.res:287]:
```
 ~isAcceptInvite=true,
```
Please check locally via removing it. I guess we can remove this then if every thing is working previously it was feature flag associated that's why I kept like that.

File: src/screens/login/HSwitchLoginFlow/HSAcceptInviteFromEmail.res [hunk 4]
```diff
@@ -7,18 +7,13 @@ let make = (~setAuthType, ~setAuthStatus) => {
...
     try {
       let url = getURL(~entityName=USERS, ~methodType=Post, ~userType=#ACCEPT_INVITE_FROM_EMAIL, ())
       let res = await updateDetails(url, body, Post, ())
       let email = res->JSON.Decode.object->Option.getOr(Dict.make())->getString("email", "")
-      let token = HyperSwitchAuthUtils.parseResponseJson(
-        ~json=res,
-        ~email,
-        ~isAcceptInvite=acceptInvite,
-      )
+      let token = HyperSwitchAuthUtils.parseResponseJson(~json=res, ~email, ~isAcceptInvite=true)
```
Comment (JeevaRamu0104) [inline src/screens/login/HSwitchLoginFlow/HSAcceptInviteFromEmail.res:16]: please check this isAcceptInvite parameter and remove it if its true always

Comment (PritishBudhiraja) [reply src/screens/login/HSwitchLoginFlow/HSAcceptInviteFromEmail.res]: Updated

File: src/screens/SwitchMerchant/SwitchMerchant.res (hunk 2 above)
Comment (PritishBudhiraja) [reply src/screens/SwitchMerchant/SwitchMerchant.res]: Updated

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**